Makefile

```diff
@@ -317,6 +320,32 @@ bundle-base: manifests kustomize operator-sdk ## Generate bundle manifests and m
 
 export CSV="./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml"
 
+.PHONY: add-console-plugin-annotation
+add-console-plugin-annotation: ## Add console-plugin annotation to the CSV
+	# add console-plugin annotation
+	$(YQ) -i '.metadata.annotations."console.openshift.io/plugins" = "[\"node-remediation-console-plugin\"]"' ${CSV}
+
+.PHONY: add-replaces-field
+add-replaces-field: ## Add replaces field to the CSV
+	# add replaces field when building versioned bundle
+	@if [ $(VERSION) != $(DEFAULT_VERSION) ]; then \
+		if [ $(PREVIOUS_VERSION) == $(DEFAULT_VERSION) ]; then \
+			echo "Error: PREVIOUS_VERSION must be set for versioned builds"; \
+			exit 1; \
+		else \
+		  	# preferring sed here, in order to have "replaces" near "version" \
+			sed -r -i "/  version: $(VERSION)/ a\  replaces: $(OPERATOR_NAME).v$(PREVIOUS_VERSION)" ${CSV}; \
+		fi \
+	fi
+
+.PHONY: bundle-ocp
+bundle-okd: yq bundle-base ## Generate bundle manifests and metadata for OKD, then validate generated files.
+	$(KUSTOMIZE) build config/manifests/okd | $(OPERATOR_SDK) generate --verbose bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	$(MAKE) add-console-plugin-annotation
+	$(MAKE) add-replaces-field
+	echo -e "\n  # Annotations for OCP\n  com.redhat.openshift.versions: \"v${OCP_VERSION}\"" >> bundle/metadata/annotations.yaml
+	ICON_BASE64="$(shell base64 --wrap=0 ./config/assets/nhc_blue.png)" \
+		$(MAKE) bundle-update
```

Comment:
Too much space?
```suggestion
$(MAKE) bundle-update
```

Reply:
`$(MAKE) bundle-update` is a continuation of the line above (see the backslash at the end of line 347), that's why it is indented

Reply:
> It was intended on purpose to signal that for us, or it is needed for how makefile works

On purpose. In Makefiles each line is a single command in its own shell, which means that setting `ICON_BASE64` value and `make bundle-update` must be on the same line, or the latter won't see the blue icon set in `ICON_BASE64`